scripts/reana-resources-k8s

```diff
@@ -46,8 +48,9 @@ populate_templates () {
         config_file_name=$(printf '%s' "$template_file_name" | sed -e "s/-template//g")
         config_file_path="$output_directory/$config_file_name"
         jinja2 "$template_file_path" "$templates_path/config.yaml" > "$config_file_path"
-        sed '/^[[:space:]]*$/d' -i "$config_file_path"
-        if [ "$(grep -c '\-\-\-' "$config_file_path")" -gt 1 ]
+        sed -i.bak '/^[[:space:]]*$/d' "$config_file_path"
+	rm "$config_file_path".bak
```

Comment:
Note the wrong indentation of the `rm` command.

BTW do you need to create backup file?  `sed -i 's/foo/bar/g' file.txt` should be enough?